docs/src/cli/choose-a-wallet.md

```diff
@@ -0,0 +1,77 @@
+# Choose a Wallet
+
+Keypairs are stored in *wallets* and wallets come in many forms. A wallet might
+be a directory on your computer's file system, a piece of paper, or a
+specialized device called a *hardware wallet*. Some wallets are easier to use
+than others.  Some are more secure than others. In this section, we'll compare
+and contrast different types of wallets and help you choose the wallet that
+best fits your needs.
+
+## File System Wallet
+
+A *file system wallet*, aka an FS wallet, is a directory in your computer's
+file system. Each file in the directory holds a keypair.
+
+### FS Wallet Security
+
+An FS wallet is the most convenient and least secure form of wallets. It is
+convenient because the keypair is stored in a simple file. You can generate as
+many keys as you'd like and trivially back them up by copying the files. If you
+are the only user of your computer and you are confident it is free of malware,
+an FS wallet is a fine solution for small amounts of cryptocurrency. If,
+however, your computer contains malware and is connected to the Internet, that
+malware may upload your keys and use them to take your tokens. Likewise,
+because the keypairs are stored on your computer as files, a skilled hacker
+with physical access to your computer may be able to access it. Using an
+encrypted hard drive, such as FileVault on MacOS, minimizes that risk.
+
+## Paper Wallet
+
+A *paper wallet* is a collection of *seed phrases* written on paper. A seed
+phrase is some number of words (typically 12 or 24) that can be used to
+regenerate a keypair on demand.
+
+### Paper Wallet Security
+
+In terms of convenience versus security, a paper wallet sits at the opposite
+side of the spectrum from an FS wallet. It is is terribly inconvenient to use,
+but offers excellent security. That high security is further amplified when
+paper wallets are used in conjunction with [offline
+signing](../offline-signing). Custody services such as [Coinbase
+Custody](https://custody.coinbase.com/) use this combination.  Paper wallets
+and custody services are an excellent way to secure a large number of tokens
+for a long period of time.
+
+## Hardware Wallet
+
+A hardware wallet is a small handheld device that stores keypairs and provides
+some interface for signing transactions.
+
+### Hardware Wallet Security
+
+A hardware wallet, such as the [Ledger hardware
+wallet](https://www.ledger.com/), offers a great blend of security and
+convenience for cryptocurrencies. It effectively automates the process of
+offline signing while retaining nearly all the convenience of an FS wallet.
+Compared to offline signing with paper wallets, which we regard as highly
+secure, some security concerns with hardware wallets:
```

Comment:
Is there some resource we can link to for the security risks/recommendations?  Seems like content better maintained by someone in that domain